daffodil-test/src/test/resources/org/apache/daffodil/usertests/SepTests.tdml

```diff
@@ -223,7 +223,7 @@
 
   <!-- Test for DAFFODIL-2499. empty strings should not be creating empty elements here. -->
   <tdml:parserTestCase name="test_sep_ssp_never_1" root="file1" model="s3"
-                       implementations="ibm">
+                       implementations="daffodil ibm">
     <!--
       This test doesn't work on Daffodil due  to DAFFODIL-2499.
```

Comment:
Since you've enabled this test, this test must be working on Daffodil now so you should remove or change this comment saying the test doesn't work on Daffodil.